Robot.java

```diff
@@ -73,13 +65,13 @@ public void robotInit() {
       camera.setExposureAuto();
     }).start();
     bosch.counterInit();
```

Comment:
#17

Reply:
Working on version with limit switches, will be addded later